monitoring/scripts/comprehensive_test.py

```
@@ -0,0 +1,752 @@
+            
+            # Test notification flow: Alertmanager -> Webhook
+            # Check if webhook is receiving notifications
+            try:
+                response = requests.get('http://localhost:5001/alerts/summary', timeout=10)
+                if response.status_code == 200:
+                    data = response.json()
+                    results['notification_flow'] = True
+                    logger.info("✅ Notification flow: Webhook service accessible")
+                else:
+                    logger.warning(f"⚠️  Notification flow: Cannot access webhook: {response.status_code}")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)